### PR TITLE
build: update golang version to 1.20

### LINKED ISSUE #117: Update golang version to 1.20
Refer: Finschia/finschia#228

### PR DESCRIPTION
**Pull request by 0Tech**
# Description

Closes #117

## Types of changes

- [ ] Bug fix (changes which fixes an issue)
- [ ] New feature (changes which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] ETC (build, ci, docs, perf, refactor, style, test)

## Checklist

- [x] I followed the [contributing guidelines](https://github.com/Finschia/wasmvm/blob/main/CONTRIBUTING.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] The PR title and commits are followed [conventional commit form](https://www.conventionalcommits.org/en/v1.0.0).

### CODE REVIEW

**Review by da1suk8 — DISMISSED** [review]

**Review by zemyblue — COMMENTED** [review]
Please add changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@
 
 ### Changes
 * chore: update libwasmvm cargo author ([#112](https://github.com/Finschia/wasmvm/pull/112))
+w
```
Comment (zemyblue) [inline CHANGELOG.md:12]: ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,9 @@
...
+* build: update golang version to 1.20 ([#118](https://github.com/Finschia/wasmvm/pull/118))
+w
```
Comment (zemyblue) [inline CHANGELOG.md:14]: ?

File: CHANGELOG.md (hunk 1 above)
Comment (0Tech) [reply CHANGELOG.md:12]: 😮

Comment (0Tech) [inline CHANGELOG.md:12]:
```suggestion
```

File: CHANGELOG.md (hunk 2 above)
Comment (0Tech) [inline CHANGELOG.md:14]:
```suggestion
```

**Review by 0Tech — COMMENTED** [review]
Remove typos.

**Review by da1suk8 — APPROVED** [review]

**Review by zemyblue — APPROVED** [review]

**Pull request merged**